src/renderer/videoJS.css

```
@@ -111,10 +111,14 @@
   font-family: VideoJS;
   font-weight: normal;
   font-style: normal;
+  margin-top: 9px;
 }
 .vjs-icon-spinner:before {
   content: "\f10b";
 }
+.vjs-button > .vjs-icon-spinner:before {
+ font-size: 1.66666666667em;
```

Minor indent issue but you should use an editor/update the editor you are using to read and enforce rules in https://github.com/FreeTubeApp/FreeTube/blob/development/.editorconfig file
See https://editorconfig.org
```suggestion
  font-size: 1.66666666667em;
```